### PR TITLE
Fixed tests

### PR DESCRIPTION
**Pull request by tstrych**

### CODE REVIEW

File: satellite_sanity_lib/rules/tests/test_sat5_frequent_osad_disconnects.py [hunk 1]
```diff
@@ -252,32 +252,31 @@ def test_get_current_time(self):
...
+            {'date': datetime.datetime(2017, 3, 15, 7, 28, 23), 'ip': '10.17.64.86', 'jid': 'osad-2b294f5c35@satellite.example.com/osad'},
+            {'date': datetime.datetime(2017, 3, 15, 7, 28, 23), 'ip': '10.17.64.86', 'jid': 'osad-ceecf22599@satellite.example.com/osad'},
+            {'date': datetime.datetime(2017, 3, 15, 7, 28, 23), 'ip': '10.17.64.86', 'jid': 'osad-b1f87862c8@satellite.example.com/osad'},
+            {'date': datetime.datetime(2017, 3, 15, 7, 28, 23), 'ip': '10.17.64.86', 'jid': 'osad-01f62470a3@satellite.example.com/osad'},
+            {'date': datetime.datetime(2017, 3, 15, 7, 28, 22), 'ip': '10.17.64.86', 'jid': 'osad-6fb988741e@satellite.example.com/osad'},
+            {'date': datetime.datetime(2017, 3, 15, 7, 8, 28), 'ip': '10.35.130.126', 'jid': 'rhn-dispatcher-sat@satellite.example.com/superclient'},
+            {'date': datetime.datetime(2017, 3, 15, 4, 38, 8), 'ip': '10.9.48.12', 'jid': 'osad-3de354aec4@satellite.example.com/osad'}]
         self.assertEqual(expected, sat5_frequent_osad_disconnects.indicator_syslog(input_data, START_DATE))
-
```
Comment (jhutar) [inline satellite_sanity_lib/rules/tests/test_sat5_frequent_osad_disconnects.py]: General rule is not to touch code which is not part of the fix/change you are doing.

If you want to remove this line, just create another pull request and we can discuss t there, although I assume this was an omission.

File: satellite_sanity_lib/rules/tests/test_sat5_frequent_osad_disconnects.py [hunk 2]
```diff
@@ -354,5 +353,5 @@ def test_match(self):
         input_data['rhn_conf'] = RHN_CONF_PG_EMB
         input_data['installed-rpms'] = INSTALLED_RPMS_570
         input_data['pgsql92_logs'] = DB92LOG_WRONG
-        expected = {'score_disconnects': 0, 'example_jid': None, 'example_ips': [], 'score_deadlocks': 1}
-        self.assertDictEqual(expected, sat5_frequent_osad_disconnects.main(input_data))
+        #expected = {'score_disconnects': 0, 'example_jid': None, 'example_ips': [], 'score_deadlocks': 1}
+        self.assertEqual(None, sat5_frequent_osad_disconnects.main(input_data)) # sat5_frequent_osad_disconnects.main(input_data) is not a dictionary so I used assertEqual not self.assertDictEqual
```
Comment (jhutar) [inline satellite_sanity_lib/rules/tests/test_sat5_frequent_osad_disconnects.py]: 1. That comment should be part of commit message, not included in the code
2. that function is called on more places in the file, why have you changed only this one? Maybe the problem is somewhere else?

**Comment by jhutar** [comment]
Sorry for wrong description in the email :-) I wanted to ask you to fix these two failures:

```
FAIL: test_indicator_syslog (test_sat5_frequent_osad_disconnects.TestSat5FrequentOsadDisconnects)
FAIL: test_indicator_syslog_wrong (test_sat5_frequent_osad_disconnects.TestSat5FrequentOsadDisconnects)
```

so please move fix for this issue out of the commit:

```
FAIL: test_match (test_sat5_frequent_osad_disconnects.TestSat5FrequentOsadDisconnects)
```

So comments just for a first part of the patch: although what you did with that year fixes the test for now, it will start failing next year again. Please try to fix the code so it will work no matter what year is it.

Also see my other two comments above.

Thank you for your work so far. I take this a bit like learning activity, so if I'm telling you something you already know, just ignore me please :-)

**Comment by tstrych** [comment]
I revert changes on test_match, add the missing end of line and change the year. 
But before this commit I also changed another file, the the hammer_ping ? I don't know if it was necessary or not,  if not I will revert it back. because i was confused from this information (# Fix "test_sat5_frequent_osad_disconnects.TestSat5FrequentOsadDisconnects" and "test_sat5_frequent_osad_disconnects.TestSat5FrequentOsadDisconnects" ), so I just try to fix tests.
So that was misunderstanding.

And of course this is learning process so I'm thankful every your idea you gave to me.

**Comment by jhutar** [comment]
Yep, sorry for that confusion and another late reply. Please rebase this pull request to only contain one commit (there is a github docs for that). Codevise, it looks good now. Ignore the other failure with hammer_ping now please.

**Comment by tstrych** [comment]
Hello, I commit once more with reverted changes in hammer_ping. I'm not
sure what I have to do, what you mean with "Please release this pull
request to only contain one commit", can you explained it more ?
I have to create new pull request just with one commit ?
Thank you

**Comment by jhutar** [comment]
Sorry, `s/release/rebase/` :-)
Check this for example: https://github.com/edx/edx-platform/wiki/How-to-Rebase-a-Pull-Request#squash-your-changes
You might need to force-push after that (see bottom of the page).

**Comment by jhutar** [comment]
:-)

**Pull request merged**